rest/data/commit_queue.go

```diff
@@ -205,8 +205,9 @@ func FindCommitQueueForProject(name string) (*restModel.APICommitQueue, error) {
 	return apiCommitQueue, nil
 }
 
-// CommitQueueRemoveItem dequeues an item from the commit queue. If the item is
-// already being tested in a batch, later items in the batch are restarted.
+// CommitQueueRemoveItem dequeues an item from the commit queue and returns the
```

Comment:
Can we rename this to something like `FindAndCommitQueueRemoveItem` just we don't have two exported functions with the same name?

Reply:
Done. I imagine this function will go away once we move to the GitHub merge queue anyways.